prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+var firstElement = languages[0];
+console.log('The first element of the array is ' + firstElement);
+
+var thirdElement = languages[2];
+console.log('The third element of the array is ' + thirdElement);
+
+var length = languages.length;
+console.log('The length of the array is ' + length);
+
+var lastElement = languages[5];
+console.log('The last element of the array is ' + lastElement);
```

Comment:
You can get the last element of any array by subtracting 1 from the length

```js
var lastElement = languages[length - 1];
```

Reply:
Hello! Thank you, I have changed it

Reply:
![Screenshot (279)](https://user-images.githubusercontent.com/86108768/149814389-47dfcb4d-4188-477b-8f93-c7d83312a008.png)